src/features/defi/providers/foxy/components/FoxyManager/Withdraw/components/Confirm.tsx

```diff
@@ -71,19 +75,30 @@ export const Confirm = ({ onNext }: StepComponentProps) => {
 
   const withdrawalFee = useMemo(() => {
     return state?.withdraw.withdrawType === WithdrawType.INSTANT
-      ? bnOrZero(bn(state.withdraw.cryptoAmount).times(state.foxyFeePercentage)).toString()
+      ? bnOrZero(
+          bn(state?.withdraw.cryptoAmount ?? '0').times(state?.foxyFeePercentage ?? '0'),
+        ).toString()
       : '0'
   }, [state?.withdraw.withdrawType, state?.withdraw.cryptoAmount, state?.foxyFeePercentage])
 
   const feeAssetBalance = useAppSelector(state =>
     selectPortfolioCryptoHumanBalanceByAssetId(state, { assetId: feeAsset?.assetId ?? '' }),
   )
 
-  if (!state || !dispatch) return null
+  const accountFilter = useMemo(() => ({ accountId: accountId ?? '' }), [accountId])
+  const accountMetaData = useAppSelector(state =>
+    selectPortfolioAccountMetadataByAccountId(state, accountFilter),
+  )
+
+  const bip44Params = accountMetaData?.bip44Params
 
-  const handleConfirm = async () => {
+  const handleConfirm = useCallback(async () => {
     try {
-      if (!state.userAddress || !rewardId || !walletState.wallet || state.loading || !api) return
+      if (
+        state?.loading ||
+        !(state?.userAddress && rewardId && walletState.wallet && api && dispatch)
```

Comment:
&& !bip44Params

Reply:
Nice catch, here we go 44ec7832650557975cc6afd98d8ff1f04fca744b